packages/i18n/src/utils.ts

```diff
@@ -11,15 +11,6 @@ export const mergeMessages = (
   ...messages: TMessageTranslations[]
 ): TMessageTranslations => Object.assign({}, ...messages);
 
-export const mapLocaleToMomentLocale = (locale: string): string => {
```

Comment:
This is not needed anymore since we now try to load the whole locale and have some fallback logic to deal with it in the loader file (`load-i18n.ts`).